### ISSUE TITLE
3D Tile Alignment

### ISSUE THREAD

**Issue by tibuch** [fmi-basel/gfriedri-em-alignment#1]
Add 3D tile alignment script.

**Comment by aloejhb** [comment]
I added a minimum working example of rendering a roughly aligned multiscale 3D volume with TensorStore.

**Comment by aloejhb** [comment]
For roughly aligning sections, this script [scripts/run_roughly_align_sections.py](https://github.com/fmi-basel/gfriedri-em-alignment/blob/08b887ada55d5d1b928d1c52a6cc66898f89b100/scripts/run_roughly_align_sections.py) on the bo-dev branch can be a starting example.
There is an issue though. I crop a small region in the center of each section for computing cross-correlation, with size specified [here](https://github.com/fmi-basel/gfriedri-em-alignment/blob/08b887ada55d5d1b928d1c52a6cc66898f89b100/config_example/align_sec.json#L10).
However, for large shifts caused by changes in sections size, e.g adding a new row of tiles, the crop_size can be too small. It would be nice to have a smart way, to first try with the small crop size, and if the shift is too large, increase the crop_size until a proper offset is found.
In that way, most sections can be aligned with small cropped region with fast speed, and a few sections with large cropped region to accommodate big shifts.

**Comment by ganctom** [comment]
The registration procedure with diffident crop sizes is a good approach, however due to the imaging issues and deformation of stitched tiles the final shift vector should be computed from shift vectors computed from multiple regions on the section pairs. The precision of rough alignment will be better.

**Comment by tibuch** [comment]
The following should work:
1. Compute coarse alignment (via the down-scaling approach) between section pairs. This can run in parallel.
2. Aggregate transformations to obtain transformation w.r.t. the first section.
